src/for_mathlib/group_completion.lean

```diff
@@ -3,11 +3,11 @@ import for_mathlib.completion
 
 universe u
 variables {α : Type*} [uniform_space α]
-variables (G : Type u) [add_comm_group G] [topological_space G] [topological_add_group G]  
+variables (G : Type u) [add_comm_group G] [topological_space G] [topological_add_group G]
 
```

Comment:
Is there a "hide all changes which are just whitespace issues when reviewing" setting in github?

Reply:
Yes there is. You can choose this in some dropdown menu at the top of the page. Alternatively, append `?w=1` to the URL.